tar/tarsnap.1-mdoc.in

```diff
@@ -1106,15 +1106,15 @@ public-domain implementation (circa November, 1987)
 was quite influential, and formed the basis of GNU tar.
 GNU tar was included as the standard system tar
 in
-FreeBSD
+.Fx
```

Comment:
Is `.Fx` available in all versions of mdoc?  I thought it was a FreeBSD extension, aka safe to use on FreeBSD but not necessarily on other systems.

Reply:
Hmm.  Libarchive uses it:
https://github.com/libarchive/libarchive/blob/master/tar/bsdtar.1#L1226
and `man bsdtar` looks fine on Ubuntu.

Also, we have `tarsnap.1-mdoc.in` as a back up in case people don't have `mdoc`.  And yes, our `mdoc2man.awk` replaces `.Fx` with `FreeBSD`.  As for "all versions of mdoc", it looks like Linux distros call it "the roff macro language of choice for BSD manual pages".  It looks like Linux pages are formatted with troff or groff or the like.

Reply:
Ok, good enough.